web/src/index.tsx

```diff
@@ -10,7 +11,8 @@ if (container == null) {
   throw new Error("Couldn't find element with id container.")
 }
 
-ReactDOM.render(<App />, container)
+const root = createRoot(container)
+root.render(<App />)
```

Comment:
Enables [concurrent renderer](https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis) in web